CHANGELOG.md

```diff
@@ -20,6 +19,9 @@
   * Add `PayPalVaultResult` type to return vault result
   * Add `PayPalVaultDelegate` to handle results from vault flow
   * Add `PayPalWebCheckoutClientError.paypalVaultResponseError` for missing or invalid response from vaulting
+* Breaking Changes
```

Comment:
Should this be targeting `next-major-version` instead of the `main` branch?

Reply:
It can be. Will the next update be a major or minor version update? Currently Android and JS have the updated shape names

Reply:
From the looks of the current CHANGELOG, all breaking changes go in major version releases. I've made the updates accordingly in CHANGELOG and the PR merges into `next-major-version`

Reply:
Thanks for making that update! Ywp, what is merged into `main` as unreleased will be released in a minor version. What is merged into `next-major-version` will contain breaking changes and will be released in a major version.

Reply:
Have you had a chance to push those changes up yet to `next-major-version`? It looks like last push to [that branch](https://github.com/paypal/paypal-ios/tree/next-major-version) was ~2 weeks ago. Happy to pair on this this afternoon or tomorrow if you'd like!